src/DocumentFormat.OpenXml/FileFormatVersions.cs

```diff
@@ -45,5 +45,10 @@ public enum FileFormatVersions
         /// Represents Microsoft Office 2021.
         /// </summary>
         Office2021 = 0x20,
+
+        /// <summary>
+        /// Represents Microsoft Microsoft 365.
+        /// </summary>
+        Microsoft365 = 0x40,
```

Comment:
This should be the final bit in the int instead of  just the next

Reply:
I meant to change that, forgot. Should be 0x80000000 I think.